compiler/testSuite/semtype/vararg-tv.bal

```diff
@@ -0,0 +1,39 @@
+//@type FT1 <> FT2
+//@type FT1 > FT1Sub1
+//@type FT1 > FT1Sub2
+//@type FT1 > FT1Sub3
+type FT1 function(int ... a);
```

Comment:
We should have some test cases here without parameter names.

Reply:
fixed